src/domains/ecommerce-app/api/ecommerce-checkout/v3/_post_sessions.xml.tpl

```diff
@@ -0,0 +1,19 @@
+<policies>
+    <inbound>
+    <rate-limit-by-key 
+    calls="150"
```

Comment:
is it possible to move this logic in base policy?  this rate limit is different considering base policy?

Reply:
It was a misprint from copy/paste action. Removed here 8e7e6f74f2349d47c06fd10e005144dd83402b80